<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setplayer</class>
 <widget class="QDialog" name="Setplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_Color">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/white (1).jpg);
font: 9pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string> Color </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>217</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/Description/Graphics/photos/white (1).jpg);
background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>51</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/white (1).jpg);
font: 9pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string> Name </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_color">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/white (1).jpg);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>229</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	color : black ;
	background-color : ;
	font: 700 13pt &quot;Segoe Print&quot;;
	border-image: 30px   url(:/Description/Graphics/photos/download7.jpg);
}

QPushButton : Hover
{
	background-color : ;
    border-radius : 40px ;
	border-image: 10px url(:/Description/Graphics/photos/download7.jpg);
}

QPushButton : pressed
{
	background-color :black ;
	color : white ;
	border-image: 10px url(:/Description/Graphics/photos/download7.jpg);

}
</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>229</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	color : black ;
	background-color : ;
	font: 700 13pt &quot;Segoe Print&quot;;
	border-image:30px ;
}

QPushButton : Hover
{
	background-color : ;
    border-radius : 40px ;
	border-image: 10px;
}

QPushButton : pressed
{
	background-color :black ;
	color : white ;
	border-image: url(:/Description/Graphics/photos/download7.jpg);

}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Age">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/white (1).jpg);
font: 9pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>  Age </string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_player_counter">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe Print&quot;;
border-image: url(:/Description/Graphics/photos/white (1).jpg);
background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string> player !</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_age">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/white (1).jpg);

color: rgb(0, 0, 0);

font: 700 italic 12pt &quot;Sitka Text&quot;;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Description/Graphics/photos/crusades-timeline-1024x512.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_Color</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>label_Name</zorder>
  <zorder>comboBox_color</zorder>
  <zorder>btn_next</zorder>
  <zorder>btn_back</zorder>
  <zorder>label_Age</zorder>
  <zorder>lb_player_counter</zorder>
  <zorder>sb_age</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
